lib/yarn-auditer.js

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright IBM All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const childProcess = require('child_process');
+const spawn = require('cross-spawn');
```

Comment:
We use `cross-spawn` instead of purely `exec` since Yarn uses JSON-lines format instead of a single JSON object. This allows us to process one _line_ at a time using `spawn`. Since `spawn` has generally better cross-platform compatibility using the `cross-spawn` library, I introduced it _just in case_.